features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/GenericSubscriptionUI.jsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import Accordion from '@material-ui/core/ExpansionPanel';
+import AccordionDetails from '@material-ui/core/ExpansionPanelDetails';
+import AccordionSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Typography from '@material-ui/core/Typography';
+import AccordionActions from '@material-ui/core/ExpansionPanelActions';
+import Button from '@material-ui/core/Button';
+import CopyToClipboard from 'react-copy-to-clipboard';
+import TextField from '@material-ui/core/TextField';
+import Alert from 'AppComponents/Shared/Alert';
+import { makeStyles } from '@material-ui/core/styles/index';
+import { FormattedMessage } from 'react-intl';
+import Grid from '@material-ui/core/Grid';
+
+const useStyles = makeStyles((theme) => (
+    {
+        bootstrapRoot: {
+            padding: 0,
+            'label + &': {
+                marginTop: theme.spacing(1),
+            },
+        },
+        bootstrapCurl: {
+            borderRadius: 4,
+            backgroundColor: theme.custom.curlGenerator.backgroundColor,
+            color: theme.custom.curlGenerator.color,
+            border: '1px solid #ced4da',
```

Comment:
Better to uses MUI [<Box/>](https://material-ui.com/system/shadows/#shadows) for these

Reply:
This is the format followed for all TextFields in the portal. Shall we leave this for consistency?
https://github.com/wso2/carbon-apimgt/blob/8f90d1304b58789a65329438fe0c08d9c336065a/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Environments.jsx#L70

Reply:
Well, then this should come from the theme @chanaka3d  ?